Add instructions on how to setup the database locally with GUI (#20)
biomage-org/iac: README.md

The permissions are configured as with any other IAM Role deployment. Make sure these
follow best practice, with each resource appropriately scoped by account ID, region,
environment, etc.

### Configurations for setting up GUI for the relational tables
The setup required to use a Relational Data Management GUI as part of your development.

#### 1. Install a GUI

> Most people use either TablePlus or pgAdmin. Use either of them, or feel free to use a different one,

Reviewer: We can also use DBeaver

Reviewer: ```suggestion
Our developers are currently using [pgAdmin](https://www.pgadmin.org/), [TablePlus](https://tableplus.com/), or [DBeaver](https://dbeaver.io/). Use either of them, or feel free to use a different one,
```

> if you have preference.

Reviewer: ```suggestion
```

#### 2. Configure the GUI

> To connect to the database in the relevant, add these fields in the connection setup window:

Reviewer: ```suggestion
To connect to the relevant database, use these fields in the connection setup window:
```

> For local development:

Reviewer: To be able to access the database in local development, `inframock` has to be running

  host: ‘localhost’
  port: 5431
  user: ‘dev_role’
  password: empty (should work)
  database: aurora_db

For staging:
  host: ‘localhost’
  port: 5432
  user: ‘dev_role’

>   password: password you get from `biomage rds token` (command available in [biomage-utils](https://github.com/biomage-org/biomage-utils))

Reviewer 2: Should we specify that for staging you need to use the -i option? `biomage rds token -i staging`
And don't we need to run also `biomage rds tunnel -i staging`?
